src/routes.json

```diff
@@ -220,6 +148,41 @@
         "title": "Community Moderator"
     },
     {
+        "name": "microworld-art",
+        "pattern": "^/microworlds/art",
+        "routeAlias": "/microworlds",
+        "view": "microworld/art/art",
+        "title": "Art"
+    },
+    {
+        "name": "microworld-fashion",
+        "pattern": "^/microworlds/fashion",
+        "routeAlias": "/microworlds",
+        "view": "microworld/fashion/fashion",
+        "title": "Fashion"
+    },
+    {
+        "name": "microworld-hiphop",
+        "pattern": "^/microworlds/hiphop",
+        "routeAlias": "/microworlds",
+        "view": "microworld/hiphop/hiphop",
+        "title": "Hip Hop Dance"
+    },
+    {
+        "name": "microworld-soccer",
+        "pattern": "^/microworlds/soccer",
+        "routeAlias": "/microworlds",
+        "view": "microworld/soccer/soccer",
+        "title": "Soccer"
+    },
+    {
+        "name": "microworlds-homepage",
```

Comment:
@mewtaylor My thought process behind this was exactly what @chrisgarrity said, I thought microworld**s** should be after microworld.